### ISSUE TITLE
Requirements for Cloud Manager Connector should also be written here.

### ISSUE THREAD

**Issue by kanitani** [NetAppDocs/occm#504]
Page: [Backing up data from an on-premises ONTAP system to the cloud](https://docs.netapp.com/us-en/occm/task_backup_from_onprem.html)

Requirements for Cloud Manager Connector should also be written here.

> Requirements
> Read the following requirements to make sure you have a supported configuration before you start backing up on-prem volumes to object storage.

**Comment by netapp-tonacki** [comment]
Hi Kanitani,  Thanks for your comment.  We do have a note in the [Limitations section](https://docs.netapp.com/us-en/occm/concept_backup_to_cloud.html#limitations) that it must be deployed in the cloud (not on premises), but you are right that it belongs in the on-prem backup section as well.  I'll add this and let you know when it is in place.

Thanks, Tom

**Comment by netapp-tonacki** [comment]
Hi Kanitani,  We have added a new section in the on-prem backup topic about [Connectors](https://docs.netapp.com/us-en/occm/task_backup_from_onprem.html#creating-or-switching-connectors) and Networking that now has this information. We have also added diagrams for backing up on-prem systems to AWS and Azure that show the Connector being deployed in the Cloud.

Thanks again for your comment.   Tom
